pom.xml

```diff
@@ -111,7 +111,7 @@
                             <mainClass>com.ericsson.eiffel.remrem.semantics.clone.PrepareLocalEiffelSchemas</mainClass>
                             <arguments>

```

Comment:
You need to also update the version in this file

Reply:
to 2.2.0

Reply:
Right, thanks. Done.